challenges-46-50.yml

```
@@ -0,0 +1,219 @@
+manual_stream:
+  question: |
+    How does the function `renderStreamMessage` work?
```

```suggestion
    How does the function `renderStreamMessage()` work?
```

And same everywhere else. The `()` just helps make it more obvious that we're talking about a function.